### DISCUSSION TITLE
Notes for January 17 meeting

### DISCUSSION THREAD

**Discussion by FranckCo** [linked-statistics/COOS#74] [General]
#### Discussion on 2022 roadmap

The following subjects are listed in the 2022 COOS activity proposal submitted to the HLG-MOS:

 - Further development based on:
   * feedback from the business experts and ontology expert
   * issues that have already been flagged with a specific "v2" label
   * More links to external model and vocabularies (DDI-CDI, SSN, DQV...)

 - Operationalization:
    * the ontology will be loaded in a RDF database and toy client applications will be developed (e.g. Model Explorer)
   * development of use cases
   * communication and promotion

#### Study of review feedback

We had feedback from [Thomas Francart](https://github.com/linked-statistics/COOS/wiki/Review-of-COOS-model---Thomas-Francart---2021-09-16) and from Jenny, Jay, Thérèse, Istat and Rosemary (issues 68 to 72). It is suggested to start with feedback from Jenny and Istat:

 - Regarding #68, typos are already integrated  (a5fe3c9), the rest has to be discussed
 - Regarding #71, Flavio started a conversation in the issue

#### Other issues

Discussion of issues still open from last year, especially those listed in [milestone 2](https://github.com/linked-statistics/COOS/milestone/2).

**Comment by ChLaaboudi** [comment]
Suggestion for more links to external model and vocabularies: 
the SDMX Working Group has an opened an action for linking SMDX and GSBPM sub-activities.
